src/specfem2D/compute_add_sources_acoustic.f90

```diff
@@ -113,35 +116,62 @@ subroutine compute_add_sources_acoustic_moving_source(potential_dot_dot_acoustic
   !local variables
   integer :: i_source,i,j,iglob,ispec
   double precision :: hlagrange
-  double precision :: xminSource,vSource,timeval,t_used
+  double precision :: xsrc,zsrc,timeval,t_used
+  real(kind=CUSTOM_REAL), dimension(NDIM,NGLLX,NGLLZ) :: sourcearray
 
   ! checks if anything to do
   if (.not. SOURCE_IS_MOVING) return
 
-  xminSource = -60.0d0 ! m
-  vSource = 60.0d0 ! m/s
-
   if (time_stepping_scheme == 1) then
     ! Newmark
     timeval = (it-1)*deltat
   else
-    call exit_MPI(myrank,'Not implemented!')
+    call exit_MPI(myrank,'Only Newmark time scheme is implemented for moving sources (2)')
+  endif
+
+  if ((myrank == 0) .and. (it < 5)) then
+    do i_source = 1,NSOURCES
+      write(IMAIN,*) ''
+      write(IMAIN,*) 'Your are using moving source capabilities. Please cite:'
```

Comment:
this is sort of a new idea to highlight citations depending on the feature used by the simulation. not sure yet, if it is the right place. it certainly should be mentioned in the doc/ manual.

maybe in the future, we could put something a the very end of the simulation, otherwise, i'm afraid that the user output will get very crowded when we do this for other features as well.

Reply:
Yeah well... I have not published a real article with this yet so it would be really painful if somebody uses it without even citing my thesis... but still I wanted to commit it so it can be used by the community. But yeah I will add a word in the manual